src/atom.rs

```diff
@@ -58,7 +58,7 @@ impl GlibPtrDefault for Atom {
 impl Uninitialized for Atom {
     #[inline]
     unsafe fn uninitialized() -> Self {
-        mem::zeroed()
+        mem::MaybeUninit::zeroed().assume_init()
```

Comment:
This does not actually bring any advantage over `mem::zeroed()`. It's still UB if all-zeroes is not a valid representation of the type.

Also see https://github.com/gtk-rs/gir/issues/808 . Why add more `Uninitialized` impls :)